src/Components/Item.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const Item =(props)=> {
+    return (
+        <div className='ItemList'>
+            <li>
+    <h3>{props.name}</h3>
+    <p>price: {props.price.toFixed(2)}</p>
+    <button>Add To Cart</button>
+    <img src={props.img} alt ={props.name}/>
+    <p>{props.description}</p>
```

Comment:
When you use the Item component inside ItemList, you pass a single prop named item.
Replace props.name with props.item.name, props.price with props.item.price, etc to fix this.